protocols/kad/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.45.1
```

Comment:
```suggestion
## 0.45.1 - unreleased
```

Reply:
Please also bump the version in `protocols/kad/Cargo.toml` and `Cargo.toml`.

Reply:
Done.